docs/notebooks/using_mclip.ipynb

```diff
@@ -0,0 +1,328 @@
+    "Currently, we only support one multilingual CLIP model, which has been made available by [open-clip](https://github.com/mlfoundations/open_clip). This model is the `xlm-roberta-base-ViT-B-32`, which has been trained on the `laion5b` dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b57559c-aa55-40ff-9d05-f061dfb01354",
+   "metadata": {},
+   "source": [
+    "## Fine-tuning\n",
+    "Now that our data has been prepared, we can start our fine-tuning run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a0cba20d-e335-43e0-8936-d926568034b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import finetuner\n",
+    "\n",
+    "run = finetuner.fit(\n",
+    "    model='xlm-roberta-base-ViT-B-32::laion5b_s13b_b90k',\n",
+    "    train_data=train_data,\n",
+    "    eval_data=eval_data,\n",
+    "    epochs=5,\n",
+    "    learning_rate=1e-6,\n",
+    "    loss='CLIPLoss',\n",
+    "    device='cuda',\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6be36da7-452b-4450-a5d5-6cae84522bb5",
+   "metadata": {},
+   "source": [
+    "You may notice that this piece of code looks very similar to the one used to fine-tune regular clip models, as shown [here](https://finetuner.jina.ai/notebooks/text_to_image/). The only real difference is the data being provided and the model being used. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "923e4206-ac60-4a75-bb3d-4acfc4218cea",
+   "metadata": {},
+   "source": [
+    "## Monitoring\n",
+    "\n",
+    "Now that we've created a run, let's see its status. You can monitor the run by checking the status - `run.status()` and - the logs - `run.logs()` or - `run.stream_logs()`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "56d020bf-8095-4a83-a532-9b6c296e985a",
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# note, the fine-tuning might takes 20~ minutes\n",
+    "for entry in run.stream_logs():\n",
+    "    print(entry)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b58930f1-d9f5-43d3-b852-5cbaa04cb1aa",
+   "metadata": {},
+   "source": [
+    "Since some runs might take up to several hours/days, it's important to know how to reconnect to Finetuner and retrieve your run.\n",
+    "\n",
+    "```python\n",
+    "import finetuner\n",
+    "\n",
+    "finetuner.login()\n",
+    "run = finetuner.get_run(run.name)\n",
+    "```\n",
+    "\n",
+    "You can continue monitoring the run by checking the status - `finetuner.run.Run.status()` or the logs - `finetuner.run.Run.logs()`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f0b81ec1-2e02-472f-b2f4-27085bb041cc",
+   "metadata": {},
+   "source": [
+    "## Evaluating\n",
+    "Currently, we don't have a user-friendly way to get evaluation metrics from the {class}`~finetuner.callback.EvaluationCallback` we initialized previously.\n",
```

Comment:
"{class}`~finetuner.callback.EvaluationCallback`"
    
    Is this correct markdown? It doesn't seem to render right.

Reply:
This section is almost identical to a section in the text_to_image notebook, I think it will render fine once it is published